lib/sheets_db/worksheet.rb

```diff
@@ -3,6 +3,8 @@
 
 module SheetsDB
   class Worksheet
+    class WorksheetColumnNotFoundError < StandardError; end
```

Comment:
Nah, I agree.  Just being overly verbose.  I agree excluding the parent class name makes sense.  Updated.